platform/installation.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Platform Installation
+summary: Guidance on the different ways to install and use the platform
+reviewed: 2019-06-15
+---
+
+The Particular Service Platform can be used in different ways.
+
+- Monitoring production systems
+- Debugging during development
+- Presenting from within Visual Studio
```

Comment:
Presenting??? What does that mean? Before scrolling down I have no idea.

I'm not sure if this enumeration is really adding any value though. What about replacing the entire intro with:

> The Particular Service Platform can either be installed to a server for use with production systems, or on a workstation during development. In addition, a portable version is available for use in samples or demonstrations.